Backend/src/main/java/lt/vu/ads/controllers/OrderController.java

```diff
@@ -0,0 +1,37 @@
+package lt.vu.ads.controllers;
+
+import lt.vu.ads.exceptions.ResourceNotFoundException;
+import lt.vu.ads.models.Order;
+import lt.vu.ads.repositories.OrderRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import java.util.List;
+
+@RestController
+@RequestMapping("/api")
+public class OrderController {
+    @Autowired
```

Comment:
Better to avoid field injection and use constructor injection

E.g. delete autowired and put "requiredargs" annotation on class